pipeline/00-ingest.R

```diff
@@ -2,36 +2,22 @@
 # 1. Setup ---------------------------------------------------------------------
 #- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 
+# NOTE: See DESCRIPTION for library dependencies and R/setup.R for
```

Comment:
I added this note at the beginning of each pipeline stage because people are used to seeing calls to `library()`, so we should direct them to the correct place.